packages/spirv_tools.rb

```diff
@@ -0,0 +1,62 @@
+# Adapted from Arch Linux spirv-tools PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/spirv-tools/trunk/PKGBUILD
+require 'package'
+
+class Spirv_tools < Package
+  description 'API and commands for processing SPIR-V modules'
+  version '2020.7'
+  license 'custom'
+  compatibility 'all'
+  source_url 'https://github.com/KhronosGroup/SPIRV-Tools/archive/v2020.7.tar.gz'
+  source_sha256 'c06eed1c7a1018b232768481184b5ae4d91d614d7bd7358dc2fe306bd0a39c6e'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/spirv_tools-2020.7-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/spirv_tools-2020.7-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/spirv_tools-2020.7-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/spirv_tools-2020.7-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '11f7b470632a94066cae7c546fc8a826c2cc34e11aab3544bb1fff64dc3627d4',
+     armv7l: '11f7b470632a94066cae7c546fc8a826c2cc34e11aab3544bb1fff64dc3627d4',
+       i686: '4a35f21e49c67571334444318560c9f182c1c31d82290927f83330ba7178f3f1',
+     x86_64: 'deaebf001fa0d2d78c24b1b7cb6a6b1c062afa88c3c8ae80d79a73db86595597'
+  })
+
+  depends_on 'spirv_headers'
+
+  def self.patch
+    # Source needs spirv-headers repo as submodule
+    @git_dir = 'external/SPIRV-Headers'
+    @git_hash = 'f88a1f98fa7a44ccfcf33d810c72b200e7d9a78a'
```

Comment:
Can we use a release tag instead of a hash?

Reply:
I tried that first. Apparently, there are lots of non-working tags when trying to get tools to work with headers for this.